apps/juxtaposition-ui/src/util.js

```diff
@@ -246,6 +246,23 @@ function updateCommunityHash(community) {
 	communityMap.set(community.olive_community_id, community.name);
 }
 
+function getReasonMap() {
+	return [
+		'Spoiler',
+		'Personal Information',
+		'Violent Content',
+		'Inappropriate/Harmful Conduct',
+		'Hateful/Bullying',
+		'Advertising',
+		'Sexually Explicit',
+		'Piracy',
+		'Inappropriate Behavior in Game',
+		'Other',
+		'Missing Images; Reach out to Jemma with post link to fix',
+		'Unknown'
+	];
```

Comment:
This won't handle Unknown correctly, as the unknown should be the "default"

You'll want to remove unknown from the array, and use a null coalescing operator to return unknown `reasonMap[report.reason] ?? "Unknown"`